unix/x0vncserver/x0vncserver.cxx

```diff
@@ -113,6 +116,22 @@ static void CleanupSignalHandler(int /*sig*/)
   caughtSignal = true;
 }
 
+static int createSystemdListeners(std::list<SocketListener*> *listeners)
+{
+#ifdef HAVE_SYSTEMD_H
+  int count = sd_listen_fds(0);
+
+  for (int i = 0; i < count; ++i) {
+      /* systemd sockets start at FD 3 */
+      listeners->push_back(new TcpListener(3 + i));
```

Comment:
There's a `SD_LISTEN_FDS_START` constant. Can't we use that to make things more clear?

Reply:
Yes. The systemd docs said "3", a constant is better than a magic number.